src/main/resources/swagger/api-docs.yaml

```diff
@@ -1091,6 +1091,30 @@ paths:
       tags:
         - Resources
 
+  /api/user/v1/{email}:
```

Comment:
I think this should be in the same top level URI as `/api/v1/invite/user/{inviteeEmail}` below (which is a little wrong anyway)

how about
`/api/users/v1/invite/{inviteeEmail}`
`/api/users/v1/info/{email}`